ios/FluentUI.Demo/FluentUI.Demo/Demos.swift

```diff
@@ -13,6 +13,7 @@ let demos: [(title: String, controllerClass: UIViewController.Type)] = [
     ("BadgeField", BadgeFieldDemoController.self),
     ("BadgeView", BadgeViewDemoController.self),
     ("Button", ButtonDemoController.self),
+	("Button (vNext)", ButtonVNextDemoController.self),
     ("Color", ColorDemoController.self),
```

Comment:
might be a good thing to add objc calling in to vnext button in ObjectiveCDemoController file as well

Reply:
I'm going to replace all usages of the button in the project with the new one in SwiftUI. This includes an instance in the ObjectiveCDemoController.